pkg/webhooks/machineset_webhook_test.go

```diff
@@ -747,6 +748,32 @@ func TestMachineSetUpdate(t *testing.T) {
 			},
 			expectedError: "providerSpec.network.devices: Required value: at least 1 network device must be provided",
 		},
+		{
+			name:         "with a modification to the selector",
+			platformType: osconfigv1.AWSPlatformType,
+			clusterID:    vsphereClusterID,
+			baseProviderSpecValue: &runtime.RawExtension{
+				Object: defaultAWSProviderSpec.DeepCopy(),
+			},
+			updateMachineSet: func(ms *machinev1.MachineSet) {
+				ms.Spec.Selector.MatchLabels["foo"] = "bar"
+			},
+			expectedError: "[spec.selector: Forbidden: selector is immutable, spec.template.metadata.labels: Invalid value: map[string]string{\"machineset-name\":\"machineset-update-abcd\"}: `selector` does not match template `labels`]",
```

Comment:
Just wondering, can we remove this  `map[string]string` thingy from this output?

Reply:
Technically we could remove it, but when there's a field that's invalid, it's typical to print out the invalid field so that users know what's wrong. Also, this is in-keeping with what happens when you do this for a deployment etc.

Reply:
>Also, this is in-keeping with what happens when you do this for a deployment etc.
That is what i was curious about. Thanks!